cmd/rule-evaluator/main.go

```diff
@@ -69,6 +69,9 @@ func main() {
 	}
 
 	exporterOptions := export.NewFlagOptions(a)
+	// TODO(danielclark): make version centralized and populated at build time.
+	exporterOptions.UserAgent = "rule-evaluator/prometheus-engine/export:v0.1.0"
```

Comment:
Per common formatting no `:`, no `v`, and no `/` except for separating component from version.